src/api_server/src/lib.rs

```diff
@@ -206,7 +206,7 @@ impl ApiServer {
 
                 let delta_us = utils::time::get_time_us(utils::time::ClockType::Monotonic)
                     - request_processing_start_us;
-                debug!("Total previous API call duration: {} us.", delta_us);
+                info!("Total previous API call duration: {} us.", delta_us);
```

Comment:
Thinking more about this, since this would be a user-visible change, should we instead set the log level in our integration tests to debug?

Reply:
will do!